frontend/src/components/custom-tools/document-manager/DocumentManager.jsx

```diff
@@ -127,7 +127,7 @@ function DocumentManager({ generateIndex, handleUpdateTool, handleDocChange }) {
 
     const requestOptions = {
       method: "GET",
-      url: `/api/v1/unstract/${sessionDetails?.orgId}/prompt-studio/file/fetch_contents?document_id=${selectedDoc?.document_id}&view_type=${viewType}&tool_id=${details?.tool_id}`,
+      url: `/api/v1/unstract/${sessionDetails?.orgId}/prompt-studio/file/${details?.tool_id}?document_id=${selectedDoc?.document_id}&view_type=${viewType}`,
```

Comment:
@johnyrahul This API is also being called inside unstract-cloud, where we get the summary content. Can you please add the change [there](https://github.com/Zipstack/unstract-cloud/blob/5adbeb24ba768a494fe4900553fd035c354930e5/frontend/src/plugins/summarize-view/SummarizeView.jsx#L40) as well?